tests/auto_shift/test_auto_shift.cpp

```diff
@@ -68,3 +68,31 @@ TEST_F(AutoShift, key_release_after_timeout) {
     run_one_scan_loop();
     VERIFY_AND_CLEAR(driver);
 }
+
+
+// test auto shift and space cadet interaction
+// press shift, press key, release shift, release key
+// the right interaction is we only get the shifted key
+// the wrong interaction is we get a bracket and a shifted key
+TEST_F(AutoShift, key_release_after_timeout) {
```

Comment:
the `key_release_after_timeout` part here needs to be unique.  At least unique to this file.  Eg:
```suggestion
TEST_F(AutoShift, auto_shift_with_space_cadet) {
```